domains/transfer-agreements/TransferAgreements.API/API/BackgroundServices/DummyMetricsWorker.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using API.Metrics;
+using Microsoft.Extensions.Hosting;
+
+namespace API.BackgroundServices
+{
+    public class DummyMetricsWorker : BackgroundService
```

Comment:
Why should this BackgroundService be merged into main?

Reply:
To have something to show for demo and to have some usage code. Could be deleted I guess